src/widgets/principles/ui/principle-card/principle-card.scss

```diff
@@ -48,6 +48,11 @@
     letter-spacing: 0;
   }
 
+  & span img {
```

Comment:
```suggestion
  span img {
```

Reply:
I used same logic as in other code. "Ugly but consistent"

fixed in: https://github.com/rolling-scopes/site/pull/483/commits/ad2cbb2fc7168ba88022942afe8f290019b89ab8